### PR TITLE
add flip descriptions

### PR DESCRIPTION
**Pull request by ayumitk**
**Pull Request Summary**

- Added the flip hover style for kpi cards
- Added the flip hover style for data catds

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**This pull request makes the following changes:**

**Changes**

- KPI cards
Screenshot 2024-01-16 at 5 44 06 PM

- Data cards
Screenshot 2024-01-16 at 5 44 19 PM

### CODE REVIEW

**Comment by Kahonnohak** [comment]
Thanks Ayumi!

Could you keep the card in dark navy for now please? Would like to see some gradients but I'd want to see with backgrounds. Text for this section are below

Build&Earn (please change the title too)
Stakers earn basic rewads, projects earn tier rewards during the period.

Basic Rewards
All stakers receive basic rewads during the B&E period.

Bonus Rewards


Also please add following 
- Remove TVL (by removing it here the look will be simple and TVL is already added to the data section)
- please use --value-small for % 
- Could you add all flips in the data section too? (all dark navy but in the future all are individually or partly changed)

**Comment by ayumitk** [comment]
@Kahonnohak Thank you for your feedback! I've updated the most of them except:

> please use --value-small for %

I would like to know more details about this.

**Comment by Kahonnohak** [comment]
> @Kahonnohak Thank you for your feedback! I've updated the most of them except:
> 
> > please use --value-small for %
> 
> I would like to know more details about this.

sorry I forgot to add the image - please use smaller size for the %

Screenshot 2024-01-16 at 23 00 48

**Review by Kahonnohak — COMMENTED** [review]
LGTM Great work!

**Comment by ayumitk** [comment]
@Kahonnohak Just FYI, all descriptions for data are placeholder

Screenshot 2024-01-16 at 5 38 30 PM

File: src/i18n/en-US/index.ts [hunk 1]
```diff
@@ -747,7 +747,7 @@ export default {
     dAppRewards: 'dApp rewards',
     done: 'Done',
     search: 'Search',
-    build: 'Build',
+    build: 'Build&Earn',
```
Comment (impelcrypto) [inline src/i18n/en-US/index.ts:750]:
```suggestion
    buildAndEarn: 'Build&Earn',
```

**Review by impelcrypto — CHANGES_REQUESTED** [review]

File: src/staking-v3/components/data/DataCard.vue [hunk 2]
```diff
@@ -27,7 +38,37 @@ export default defineComponent({
...
-.wrapper--data {
+.wrapper--data-card {
+  height: 150px;
+}
+
+.card--inner {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  transition: transform 0.3s;
```
Comment (bobo-k2) [inline src/staking-v3/components/data/DataCard.vue:49]: Maybe to slow down animation a bit.

**Review by bobo-k2 — APPROVED** [review]

File: src/i18n/en-US/index.ts (hunk 1 above)
Comment (ayumitk) [reply src/i18n/en-US/index.ts]: @impelcrypto fixed in [30fc231](https://github.com/AstarNetwork/astar-apps/pull/1129/commits/30fc231b1dae22f4fc7ac1a2c09b96916d50f9cf)

**Review by Kahonnohak — APPROVED** [review]
LGTM

**Review by impelcrypto — APPROVED** [review]

**Pull request merged**